addon/instance-initializers/uikit.js

```diff
@@ -0,0 +1,7 @@
+import UIkit from "uikit";
+
+export function initialize(appInstance) {
+  UIkit.container = appInstance.rootElement;
```

Comment:
Will this work for e.g. camac-ng, is the container id set through the rootElement there as well?